wagtailmedia/views/chooser.py

```diff
@@ -5,15 +5,17 @@
 from wagtail.admin.modal_workflow import render_modal_workflow
 from wagtail.admin.utils import PermissionPolicyChecker
 from wagtail.core.models import Collection
-from wagtail.utils.pagination import paginate
 
 from wagtailmedia.models import get_media_model
 from wagtailmedia.permissions import permission_policy
+from wagtailmedia.utils import paginate
 
 if WAGTAIL_VERSION < (2, 5):
     from wagtail.admin.forms import SearchForm
+    pagination_template = "wagtailadmin/shared/pagination_nav.html"
 else:
     from wagtail.admin.forms.search import SearchForm
+    pagination_template = "wagtailadmin/shared/ajax_pagination_nav.html"
```

Comment:
Is there a reason not to use "wagtailadmin/shared/pagination_nav.html" in both cases?

Reply:
Hey Dan!

Using `pagination_nav.html` throws a `RemovedInWagtail27Warning`.

Any other ideas?

Reply:
It is odd as the warning comes from using `wagtail.utils.pagination` as per https://github.com/wagtail/wagtail/pull/5128/files#diff-141f526e4edead2de767c3662862a42aR8

and `pagination_nav.html` has just an extra block https://github.com/wagtail/wagtail/blob/master/wagtail/admin/templates/wagtailadmin/shared/pagination_nav.html#L12-L14 🤔

Reply:
The reason it throws the warning is https://github.com/torchbox/wagtailmedia/blob/master/wagtailmedia/templates/wagtailmedia/chooser/results.html#L17

Reply:
so the implementation is 👍